aspnet/web-api/overview/older-versions/self-host-a-web-api.md

```diff
@@ -30,15 +30,15 @@ Start Visual Studio and select **New Project** from the **Start** page. Or, from
 
 In the **Templates** pane, select **Installed Templates** and expand the **Visual C#** node. Under **Visual C#**, select **Windows**. In the list of project templates, select **Console Application**. Name the project &quot;SelfHost&quot; and click **OK**.
 
-![](self-host-a-web-api/_static/image1.png)
+![Screenshot of the New Project dialog box showing the Visual C# item in the folder list and the Windows Forms Application item highlighted in gray.](self-host-a-web-api/_static/image1.png)
```

Comment:
Sort of feels like a run on sentence. Maybe a comma after 'item' and then add 'which is' before the phrase 'highlighted in grey'